openwisp_users/api/mixins.py

```diff
@@ -135,8 +137,11 @@ def filter_fields(self):
                 )
                 continue
             try:
+                conditions = Q(**{self.organization_lookup: organization_filter})
```

Comment:
`conditions`, can we change the varible name to something like `org_filter` or `org_filter_condiitions` or something like that, `conditions` alone sounds out of place!

Reply:
`conditions` is fine, those are the conditions of the filter so it's good enough for me